_data/apis.json

```diff
@@ -406,7 +406,7 @@
             "https://www.acolad.com/en/technology/machine-translation.html"
         ],
         "languages": [],
-        "self-serve": false
+        "self-serve": true
```

Comment:
I think it's still false, no?

On their page it says "Get in touch", it doesn't just let you try it out.